sdk/daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v1/ScriptF.scala

```diff
@@ -878,37 +885,41 @@ object ScriptF {
 
   private def parseAllocParty(ctx: Ctx, v: SValue): Either[String, AllocParty] = {
     def convert(
-        idHint: String,
+        requestedName: String,
+        givenHint: String,
         participantName: SValue,
         stackTrace: Option[SValue],
         continue: SValue,
     ) =
       for {
         participantName <- Converter.toParticipantName(participantName)
         stackTrace <- toStackTrace(ctx, stackTrace)
+        idHint <- Converter.toPartyIdHint(givenHint, requestedName, ctx.random)
```

Comment:
Maybe the order of the arguments could be realigned between `convert` and `Converter.toPartyIdHint`. As both are strings, it is a problem waiting to happen.

Reply:
Done